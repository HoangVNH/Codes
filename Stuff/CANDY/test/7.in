10
2
XX
4
VVDD
10
XXXXXXXDDD
8
DDDXXXVX
10
XXXVVVDDVV
2
VV
5
XXVVX
2
VX
1
V
4
XXXV
