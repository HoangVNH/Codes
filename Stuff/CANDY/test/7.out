0
2
9
7
9
0
4
1
0
3
